charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       serviceAccountName: {{ .Values.serviceAccount.name }}
       {{- if .Values.additionalVolumes }}
       volumes:
-{{ toYaml .Values.additionalVolumes | indent 6 }}
+{{toYaml .Values.additionalVolumes | nindent 8 }}
```

Comment:
```suggestion
{{ toYaml .Values.additionalVolumes | indent 8 }}
```

Reply:
Be consistent with the existing format of the file

Reply:
This PR closed